.github/workflows/pull_request_check.yaml

```diff
@@ -0,0 +1,14 @@
+name: Pull Request Check
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+jobs:
+  pull_request_check:
+    name: pull request check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: 'nearform-actions/github-action-check-linked-issues'
```

Comment:
I think this should be nearform/github-action-check-linked-issues@v1.4.5 without quotes